.codeclimate.yml

```diff
@@ -22,3 +22,4 @@ exclude_patterns:
   - "vendor/"
   - "_example/webinar.go"
   - "_example/add_group_member.go"
+  - "_example/users/*.go"
```

Comment:
Please add a newline to the end of the file (not a blankline, just a newline) - it should be a setting in your editor